contracts/contracts/interfaces/IERC20Details.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
I think we can delete this file

Reply:
Yes, it can be deleted. I was using it in a Hardhat test as I needed to access an asset's decimals but that is no longer being used.